tests/integration/profile_test.py

```diff
@@ -61,6 +61,22 @@
     "eth1",
 ]
 
+NMCLI_CLONE_CON_CMD = [
```

Comment:
I think these are too specific to be placed here.
It will fit well under the relevant place it is used.

(unless you think you will need this soon)

Reply:
You are right, I moved it to the function that is using it. Thanks!